### PR TITLE
Gorthage Truck - New PCB

### PR DESCRIPTION
**Pull request by jpuerto96**
## Description

This PR adds a new keyboard (gorthage truck).

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/gorthage_truck/readme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# gorthage_truck
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [s8erdude](https://github.com/jpuerto96)
+* Hardware Supported: *Gorthage Truck PCB*
```
Comment (fauxpark) [inline keyboards/gorthage_truck/readme.md:6]:
```suggestion
A short description of the keyboard/project

* Keyboard Maintainer: [s8erdude](https://github.com/jpuerto96)
* Hardware Supported: Gorthage Truck PCB
```
Please fill this out.

File: keyboards/gorthage_truck/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    _BASE,
+    _FN
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [_BASE] = LAYOUT_3u(
+KC_TAB, 	KC_Q, 		KC_W, 		KC_E, KC_R, KC_T, KC_Y, 	KC_U, 		KC_I, 		KC_O, KC_P, KC_BSPC, KC_MINUS,                KC_1, KC_2, KC_3, \
+KC_SLCK, 	KC_A, 		KC_S, 		KC_D, KC_F, KC_G, KC_H, 	KC_J, 		KC_K, 		KC_L, KC_SCLN, KC_QUOT, KC_EQUAL,            KC_4, KC_5, KC_6, \
+KC_NLCK, 	KC_Z, 		KC_X, 		KC_C, KC_V, KC_B, KC_N, 	KC_M, 		KC_COMM, 	KC_DOT, KC_ENT, KC_SLSH, KC_GRAVE, KC_UP,  KC_7, KC_8, KC_9, \
```
Comment (fauxpark) [inline keyboards/gorthage_truck/keymaps/default/keymap.c:29]:
```suggestion
KC_TAB, 	KC_Q, 		KC_W, 		KC_E, KC_R, KC_T, KC_Y, 	KC_U, 		KC_I, 		KC_O, KC_P, KC_BSPC, KC_MINUS,                KC_1, KC_2, KC_3,
KC_SLCK, 	KC_A, 		KC_S, 		KC_D, KC_F, KC_G, KC_H, 	KC_J, 		KC_K, 		KC_L, KC_SCLN, KC_QUOT, KC_EQUAL,            KC_4, KC_5, KC_6,
KC_NLCK, 	KC_Z, 		KC_X, 		KC_C, KC_V, KC_B, KC_N, 	KC_M, 		KC_COMM, 	KC_DOT, KC_ENT, KC_SLSH, KC_GRAVE, KC_UP,  KC_7, KC_8, KC_9,
```

File: keyboards/gorthage_truck/keymaps/7u/keymap.c [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    _BASE,
+    _FN
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [_BASE] = LAYOUT_7u(
+KC_TAB, 	KC_Q, 		KC_W, 		KC_E, KC_R, KC_T, KC_Y, 	KC_U, 		KC_I, 		KC_O, KC_P, KC_BSPC, KC_MINUS,                KC_1, KC_2, KC_3, \
+KC_LCTL, 	KC_A, 		KC_S, 		KC_D, KC_F, KC_G, KC_H, 	KC_J, 		KC_K, 		KC_L, KC_SCLN, KC_QUOT, KC_EQUAL,            KC_4, KC_5, KC_6, \
+KC_LCAP, 	KC_Z, 		KC_X, 		KC_C, KC_V, KC_B, KC_N, 	KC_M, 		KC_COMM, 	KC_DOT, KC_ENT, KC_SLSH, KC_GRAVE, KC_UP,  KC_7, KC_8, KC_9, \
```
Comment (fauxpark) [inline keyboards/gorthage_truck/keymaps/7u/keymap.c:29]:
```suggestion
KC_TAB, 	KC_Q, 		KC_W, 		KC_E, KC_R, KC_T, KC_Y, 	KC_U, 		KC_I, 		KC_O, KC_P, KC_BSPC, KC_MINUS,                KC_1, KC_2, KC_3,
KC_LCTL, 	KC_A, 		KC_S, 		KC_D, KC_F, KC_G, KC_H, 	KC_J, 		KC_K, 		KC_L, KC_SCLN, KC_QUOT, KC_EQUAL,            KC_4, KC_5, KC_6,
KC_LCAP, 	KC_Z, 		KC_X, 		KC_C, KC_V, KC_B, KC_N, 	KC_M, 		KC_COMM, 	KC_DOT, KC_ENT, KC_SLSH, KC_GRAVE, KC_UP,  KC_7, KC_8, KC_9,
```

File: keyboards/gorthage_truck/keymaps/10u/keymap.c [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    _BASE,
+    _FN
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [_BASE] = LAYOUT_10u(
+KC_TAB, 	KC_Q, 		KC_W, 		KC_E, KC_R, KC_T, KC_Y, 	KC_U, 		KC_I, 		KC_O, KC_P, KC_BSPC, KC_MINUS,                KC_1, KC_2, KC_3, \
+KC_LCTL, 	KC_A, 		KC_S, 		KC_D, KC_F, KC_G, KC_H, 	KC_J, 		KC_K, 		KC_L, KC_SCLN, KC_QUOT, KC_EQUAL,            KC_4, KC_5, KC_6, \
+KC_LCAP, 	KC_Z, 		KC_X, 		KC_C, KC_V, KC_B, KC_N, 	KC_M, 		KC_COMM, 	KC_DOT, KC_ENT, KC_SLSH, KC_GRAVE, KC_UP,  KC_7, KC_8, KC_9, \
```
Comment (fauxpark) [inline keyboards/gorthage_truck/keymaps/10u/keymap.c:29]:
```suggestion
KC_TAB, 	KC_Q, 		KC_W, 		KC_E, KC_R, KC_T, KC_Y, 	KC_U, 		KC_I, 		KC_O, KC_P, KC_BSPC, KC_MINUS,                KC_1, KC_2, KC_3,
KC_LCTL, 	KC_A, 		KC_S, 		KC_D, KC_F, KC_G, KC_H, 	KC_J, 		KC_K, 		KC_L, KC_SCLN, KC_QUOT, KC_EQUAL,            KC_4, KC_5, KC_6,
KC_LCAP, 	KC_Z, 		KC_X, 		KC_C, KC_V, KC_B, KC_N, 	KC_M, 		KC_COMM, 	KC_DOT, KC_ENT, KC_SLSH, KC_GRAVE, KC_UP,  KC_7, KC_8, KC_9,
```

File: keyboards/gorthage_truck/config.h [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+// LEFT -> RIGHT
+// D5, D3, D2
+#define LED_NUM_LOCK_PIN D5
+#define LED_CAPS_LOCK_PIN D3
+#define LED_SCROLL_LOCK_PIN D2
+
+#define ENCODERS_PAD_A { F5 }
+#define ENCODERS_PAD_B { F6 }
+
+#define ENCODER_RESOLUTION 4
```
Comment (fauxpark) [inline keyboards/gorthage_truck/config.h:62]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by jpuerto96** [comment]
Changes have been made. Please review when possible @fauxpark. Thanks!

File: keyboards/gorthage_truck/readme.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# gorthage_truck
+
+A short description of the keyboard/project
+
```
Comment (fauxpark) [inline keyboards/gorthage_truck/readme.md:4]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**